public/app/features/dashboard/components/ShareModal/ShareEmbed.tsx

```diff
@@ -81,24 +80,15 @@ export class ShareEmbed extends PureComponent<Props, State> {
         <div className="share-modal-header">
           <Icon name="link" className="share-modal-big-icon" size="xxl" />
           <div className="share-modal-content">
-            <div className="gf-form-group">
-              <Switch
-                labelClass="width-12"
-                label="Current time range"
-                checked={useCurrentTimeRange}
-                onChange={this.onUseCurrentTimeRangeChange}
-              />
-              <Switch
-                labelClass="width-12"
-                label="Template variables"
-                checked={includeTemplateVars}
-                onChange={this.onIncludeTemplateVarsChange}
-              />
-              <div className="gf-form">
-                <label className="gf-form-label width-12">Theme</label>
-                <Select width={10} options={themeOptions} value={selectedTheme} onChange={this.onThemeChange} />
-              </div>
-            </div>
+            <InlineField labelWidth={24} label="Current time range">
+              <Switch value={useCurrentTimeRange} onChange={this.onUseCurrentTimeRangeChange} />
```

Comment:
I think you'd be able to pass `id` to switch so it will be connected to the label.